language/en/default.php

```diff
@@ -96,7 +96,8 @@
     'alert_coupon_not_applied' => 'Your coupon can not be applied on orders below %s.',
     'alert_coupon_maximum_reached' => 'Maximum number of redemption for the coupon has been reached.',
     'alert_coupon_order_restriction' => 'Your coupon can not be applied to <b>%s</b> orders',
-    'alert_menu_not_within_mealtime' => '%s is available for %s ONLY between (%s - %s)',
+    'alert_menu_not_within_mealtime' => '%s is available for %s',
```

Comment:
Would rename this to `alert_menu_not_within_mealtimes` so it doesn't conflict with stored lang strings causing errors with `sprintf`

Reply:
No problem - updates